assets/helpers/flashSale.js

```diff
@@ -135,6 +135,64 @@ const Sales: Sale[] = [
 
     },
   },
+  {
+    subscriptionProduct: 'Paper',
+    activeRegions: ['GBPCountries'],
+    startTime: new Date(2019, 0, 5).getTime(), // 4 Jan 2019
+    endTime: new Date(2019, 1, 3).getTime(), // 3 Feb 2019
```

Comment:
Actually I've just looked at the card and realised the date is wrong. It should be 4th of Feb not 3rd because the time it will finish is 0:00 in the morning.

Reply:
Thanks Rupert. I will change the end date.